mgmtd/mgmt_vty.c

```diff
@@ -349,6 +349,50 @@ DEFPY(mgmt_save_config,
 	return CMD_SUCCESS;
 }
 
+DEFPY(mgmt_copy_config,
+      mgmt_copy_config_cmd,
+      "[no$no] mgmt copy-config running startup [always$always]",
+      NO_STR
+      MGMTD_STR
+      "Save configuration from datastore\n"
+      "From Running datastore\n"
+      "To Startup Database\n"
+      "Always save configuration from running to startup automatically\n")
```

Comment:
Why is this not using YANG and northbound code for config? Of all the components that need to be leading in the proper use of the new northbound and YANG, mgmtd is at the top of the list.

Reply:
It is using Libyang code internally in mgmt_ds_dump_tree().